packages/headless/src/controllers/commerce/context/headless-context.ts

```diff
@@ -60,6 +67,12 @@ export interface Context extends Controller {
    */
   setView(view: View): void;
 
+  /**
+   * Sets the user.
+   * @param user - The new user.
+   */
+  setUser(user: User): void;
```

Comment:
should we name the method `setLocation` instead? And then the property in the component state could also be called that.

I don't mind the naming in the engine state because it's opaque, but it feels a bit odd to me that we're calling the method "setUser" when all it does is set a latitude and longitude!

Reply:
Applied this in https://github.com/coveo/ui-kit/pull/4572/commits/a95e7d68370e0c7b73059063a3d1ac6c8568eee5